dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/dataapproval/DataApprovalServiceCategoryOptionGroupTest.java

```diff
@@ -605,7 +605,7 @@ private List<String> getApprovalsExtended( User mockUserService, DataApprovalWor
     {
         setUser( mockUserService );
         List<DataApprovalStatus> approvals = dataApprovalService.getUserDataApprovalsAndPermissions( workflow, period,
-            orgUnit, ouFilter, mechanismCategoryCombo, aoc );
+            orgUnit, ouFilter, mechanismCategoryCombo, aoc == null ? null : Set.of( aoc ) );
```

Comment:
Ah, that would create a `Set` with a null element, but the trouble is that won't work at the lower levels which are expecting `null` instead of the `Set` when there are no elements.